rocketpy/simulation/flight.py

```diff
@@ -1296,6 +1347,17 @@ def udot_rail1(self, t, u, post_processing=False):
         else:
             ax, ay, az = 0, 0, 0
 
+        if post_processing:
+            # Use u_dot post processing code for forces, moments and env data
+            self.u_dot_generalized(t, u, post_processing=True)
+            # Save feasible accelerations
+            self.ax_list[-1] = [t, ax]
+            self.ay_list[-1] = [t, ay]
+            self.az_list[-1] = [t, az]
+            self.alpha1_list[-1] = [t, 0]
+            self.alpha2_list[-1] = [t, 0]
+            self.alpha3_list[-1] = [t, 0]
+
```

Comment:
What about other lists? R1_list, R2_list, R3_list, M1_list, M2_list, M3_list, pressure_list, density_list, dynamic_viscosity_list,speed_of_sound_list,wind_velocity_x_list, wind_velocity_y_list

I understand the code might be working, but these vectors will have different lengths. Could that be a problem somehow? Did you think about it?

Reply:
Btw in the future the atmospheric properties will no longer be calculated by post-process. Since we only need the z coordinate use the environment class to calculate all the atmospheric properties, we can be creating different cached_property instead of using appending() within the simulation loop.

Reply:
> What about other lists? R1_list, R2_list, R3_list, M1_list, M2_list, M3_list, pressure_list, density_list, dynamic_viscosity_list,speed_of_sound_list,wind_velocity_x_list, wind_velocity_y_list

They are appended to inside `u_dot_generalized` with the correct values since it has `post_processing=True`. The accelerations are also append to there, but those values are incorrect since it is not considering the launch rail reaction forces. The correct acceleration values comes from `udot_rail1`, and so the last item in the acceleration arrays are overwritten.

The vectors end up being the same length